tests/test_jump

```diff
@@ -0,0 +1,43 @@
+import pyglet
+import pytest
+from unittest.mock import Mock
+from tempus_fugit_minecraft.window import Window
+
+@pytest.fixture(scope="class")
+def window():
+    yield Window()
+
+class TestJump:
+    @pytest.fixture(autouse=True)
+    def teardown(self, window):
+        window.model.player.jump_speed = 5
+
+    #Issue #39 Test Q key down to increase jump
+    def test_Q_key(self, window): 
+        assert window.model.player.jump_speed == 5
+
+        window.on_key_press(pyglet.window.key.Q, Mock())
+        assert window.model.player.jump_speed == 10
+
+        window.on_key_press(pyglet.window.key.Q, Mock())
+        assert window.model.player.jump_speed == 15
+
+        for _ in range(0, 9):
+            window.on_key_press(pyglet.window.key.Q, Mock())
+        assert window.model.player.jump_speed == 20
```

Comment:
These tests will fail currently. Right now the player jump_speed is set to be the sqrt(40), (more specifically:`sqrt(2 * GRAVITY * MAX_JUMP_HEIGHT) == sqrt(2 * 20 * 1) == sqrt(40)`) which is roughly 6. So, adding 5 to these won't make 10, 15 or 20, it will make 11, 16, and 21 (also roughly).

Also, these tests don't actually run since this isn't a `.py` file.

Reply:
> These tests will fail currently. Right now the player jump_speed is set to be the sqrt(40), (more specifically:`sqrt(2 * GRAVITY * MAX_JUMP_HEIGHT) == sqrt(2 * 20 * 1) == sqrt(40)`) which is roughly 6. So, adding 5 to these won't make 10, 15 or 20, it will make 11, 16, and 21 (also roughly).
> 
> Also, these tests don't actually run since this isn't a `.py` file.

file Fixed with .py extension, and test pass, Done!